tests/datadriven/test_rna_quantification.py

```diff
@@ -175,3 +180,22 @@ def testSearchExpressionLevelsWithFeatureIds(self):
         self.assertEqual(
             _expressionTestData["num_expression_entries"],
             len(expressionLevels))
+
+    def testLoadRsemData(self):
+        """
+        Test ingest of rsem data.
+        """
+        tempDir = tempfile.mkdtemp(prefix="ga4gh_rna_quant",
+                                   dir=tempfile.gettempdir())
+        dbName = os.path.join(tempDir, "rnaQuantDB")
+        storeDb = rnaseq2ga.RnaSqliteStore(dbName)
+        storeDb.createTables()
+
+        testTsvFile = os.path.join(
+                            paths.testDataDir,
+                            "datasets/dataset1/rnaQuant/rsem_test_data.tsv")
```

Comment:
This test is really just guaranteeing the whole this doesn't go belly up, which is a great improvement!